src/CLASS2/pair_lj_class2_coul_long.h

```diff
@@ -31,7 +29,7 @@ class PairLJClass2CoulLong : public Pair {
   virtual void compute(int, int);
   virtual void settings(int, char **);
   void coeff(int, char **);
-  virtual void init_style();
```

Comment:
please do not remove the virtual keyword here. it is required for the KOKKOS version.